File: BBIS.Application/DTOs/Patient/PatientDto.cs
namespace BBIS.Application.DTOs.Patient
{
    public class PatientDto
    {
        public Guid? PatientId { get; set; }    
        public string BloodType { get; set; }
        public string Rh { get; set; }
        public string PatientIdNo { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string MiddleName { get; set; }
        public string Gender { get; set; }
        public string CivilStatus { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string ContactNumber { get; set; }
        public string Address { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/Patient/PatientDto.ts
export class PatientDto {
    PatientId: Guid | null = null;
    BloodType: string = '';
    Rh: string = '';
    PatientIdNo: string = '';
    LastName: string = '';
    FirstName: string = '';
    MiddleName: string = '';
    Gender: string = '';
    CivilStatus: string = '';
    DateOfBirth: Date | null = null;
    ContactNumber: string = '';
    Address: string = '';
}
